app/Platform/EventServiceProvider.php

```diff
@@ -33,20 +34,21 @@ class EventServiceProvider extends ServiceProvider
 {
     protected $listen = [
         AchievementCreated::class => [
+            DispatchUpdateGameMetricsJob::class, // dispatches GameMetricsUpdated
         ],
         AchievementPublished::class => [
             DispatchUpdateGameMetricsJob::class, // dispatches GameMetricsUpdated
-            // TODO DispatchUpdateDeveloperContributionYieldJob::class,
```

Comment:
Is there some reason this was TODO'd?

Reply:
Probably from line 1 in this comment: https://discord.com/channels/476211979464343552/1026595325038833725/1159436046837223515

Reply:
https://github.com/RetroAchievements/RAWeb/pull/1874#discussion_r1347230085

Reply:
Ok. I'll put these back and comment out the related assertions.